doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -321,6 +338,8 @@ Requests to the environment
 
             Search (?x * _ + ?x * _)%Z outside OmegaLemmas.
 
+   .. example of cmdv that should be a cmd
```

Comment:
Not sure.  I think SearchAbout is indeed a (deprecated) variant of Search.

Reply:
Deprecated since 8.5.  I wonder what we're waiting for.

Reply:
Feel free to open the PR removing it.